doc/qtcreator/src/mer/creator-deploying-mer.qdoc

```diff
@@ -29,18 +29,54 @@
 
     \title Deploying Applications to Sailfish OS Devices
 
-    You can specify settings for deploying applications to Sailfish OS devices in the
-    project .pro file. You can view the settings in the \gui {Run Settings}.
+    You can specify settings for deploying applications to Sailfish OS devices
+    in the project .pro file. You can view the settings in the \gui Projects
+    mode, in \gui {Run Settings}.
 
-    When you run the application on a Sailfish OS device or on the emulator, \QC
-    deploys the application as specified by the deploy steps.
+    \section1 Deployment Methods
 
-    \section1 Copying Binary Files
+    \image creator-sailfish-deploy-settings.png
 
-    \image creator-mer-deploy-settings.png
+    Three deployment methods are available.
 
-    The \gui {Deploy by Copying Binaries} step determines that \QC uses the
-    rsync file copying tool to transfer the files to the device or to the
-    emulator.
+    \list
+
+    \li Deploy As RPM Package
+
+        This is the default method. An RPM package will be build and
+        immediately deployed to the device.
+
+    \li Deploy By Copying Binaries
+
+        This method allows speeding up frequent deployments during development
+        by skipping the RPM build step.
+
+        It comes at a cost though. In order to avoid overwritting system files
+        accidentaly, the application will be deployed under the \c
+        /opt/sdk/<project-name> prefix when this method is used. This will
+        inevitably affect visibily of files that are expected to be installed
+        under fixed locations. The common approach to deal with this is to
+        deploy with RPM package initially and every time any of the affected
+        files change.
+
+        For QML based application development a more convenient option exists
```

Comment:
Maybe "For developing the QML parts of the application..."

Reply:
Tried something else.